clifford/_layout.py

```diff
@@ -589,11 +589,11 @@ def _hitzer_inverse(self):
         """
         Performs the inversion operation as described in the paper :cite:`Hitzer_Sangwine_2017`
         """
-        tot = np.sum(self.sig != 0)
+        tot = len(self.sig)
         @_numba_utils.njit
         def hitzer_inverse(operand):
             if tot == 0:
-                numerator = operand.layout.scalar
+                numerator = 1 + 0*operand
```

Comment:
What motivates this change?

Reply:
Currently LayoutType does not provide `.scalar` so the JIT fails unless you do something like this

Reply:
How was this never a problem before? I guess the code path got optimized out!

Reply:
I think before we never tested it with Cl(0,0,0) it never hit a case where `tot == 0`!

Reply:
Leaving this conversation unresolved so I remember to open an issue about making this less ugly.

Reply:
#385